### PR TITLE
[1.2] Fix onnx 1.6.0 for pb 3.11.x

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

This fixes ONNX 1.6.0 builds when using protobuf 3.11 or newer.
TensorFlow 2.4 and PyTorch 1.7 are moving to a new protobuf, but we want to stay with ONNX 1.6 for now since TensorRT is still there.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by bnemanich — APPROVED** [review]

**Comment by bnemanich** [comment]
@jayfurmanek Does this still need to be merged in?

**Comment by jayfurmanek** [comment]
no. We reverted back to pb 3.9 for this release. Perhaps will be needed in the future.

**Comment by jayfurmanek** [comment]
We'll use this for our 1.2 release which will be mostly done to increment protobuf to newer across the board.

**Comment by jayfurmanek** [comment]
https://github.com/open-ce/open-ce-environments/pull/18

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,11 @@ package:
 source:
   git_url: "https://github.com/onnx/onnx.git"
   git_tag: {{ git_tag }}
+  patches:
+    - 0101-python_out-does-not-recognize-dllexport_decl.patch
```
Comment (bnemanich) [inline recipe/meta.yaml:14]: Could add something like: `# [protobuf > 3.9]` for this patch.

**Pull request merged**